autosubmit/job/job_list_persistence.py

```diff
@@ -140,8 +142,14 @@ class JobListPersistenceDb(JobListPersistence):
         "wrapper_type",
     ]
 
-    def __init__(self, persistence_path, persistence_file):
-        self.db_manager = DbManager(persistence_path, persistence_file, self.VERSION)
+    def __init__(self, expid: str):
+        options = {
+            "root_path": str(Path(BasicConfig.LOCAL_ROOT_DIR, expid, "pkl")),
+            "db_name": f"job_list_{expid}",
+            "db_version": self.VERSION
```

Comment:
```suggestion
            "db_version": self.VERSION,
            "schema": expid
```

I think we forgot to add the schema here :point_up: 

About the changes in the tests, they look way simpler than before. Nice change! :raised_hands: